packages/experimental/Menu/src/MenuPopover/MenuPopover.tsx

```diff
@@ -21,7 +22,9 @@ export const MenuPopover = stagedComponent((props: MenuPopoverProps) => {
         directionalHint={state.directionalHint}
         doNotTakePointerCapture={state.doNotTakePointerCapture}
       >
-        {children}
+        <View onMouseEnter={state.onMouseEnter} onMouseLeave={state.onMouseLeave}>
```

Comment:
This will change to adaptable view, gotta make this slots first.

Reply:
Not a pressable/View as pressable as I don't care about pressable state, just tracking mouse in/out

Reply:
Decided to just use React.createElement, slots/compose seemed overkill